src/Raven.Server/Web/System/AdminPrometheusMonitoringHandler.cs

```diff
@@ -138,6 +138,12 @@ private async Task WriteServerMetricsAsync(MetricsProvider provider, Stream resp
                         new Size(serverMetrics.Memory.TotalSwapUsageInMb, SizeUnit.Megabytes).GetValue(SizeUnit.Bytes));
                     WriteGaugeWithHelp(writer, "Server working set swap usage", "memory_working_set_swap_usage_bytes",
                         new Size(serverMetrics.Memory.WorkingSetSwapUsageInMb, SizeUnit.Megabytes).GetValue(SizeUnit.Bytes));
+                    WriteGaugeWithHelp(writer, "Available memory for processing", "available_memory_for_processing", 
```

Comment:
Shouldn't this be `available_memory_for_processing_bytes` ?

Reply:
@Lwiel Please check names against this: https://prometheus.io/docs/practices/naming/